src/pages/components/data-table/style.mdx

```diff
@@ -290,3 +306,87 @@ sizes.
 <Caption>
   Structure and spacing measurements for batch action bar | px/rem
 </Caption>
+
+## AI presence
+
+The following are the unique styles applied to the component when the AI slug is
+present. Unless specified, all other tokens in the component remain the same as
+the non-AI variant.
+
+More information about AI style elements is coming soon.
+
+#### Entire table
+
```

Comment:
Oh because the formatting is weird with H4 and tables we have to manually add in a line break between them. Just add `<br />` in space.
<img width="776" alt="image" src="https://github.com/carbon-design-system/carbon-website/assets/11670886/5d43c4bc-7acf-48aa-a827-93f3f3d97ab1">


```suggestion
<br />
```

Reply:
Oh, i didn't know that! Thanks, will keep it in mind for future doc work 🥳

Reply:
<br/>